README.md

```diff
@@ -11,24 +9,27 @@ Maintaining its lifecycle places a burden on the operational team.
 
 ## Getting started
 There are two deployment options available, you can:
-- [run it on Google Kubernetes Engine](https://www.dynatrace.com/support/help/shortlink/deploy-k8) or 
-- [deploy a Google Cloud Function](https://www.dynatrace.com/support/help/shortlink/deploy-gcp)
+- [Metrics and/or logs] [run it on Google Kubernetes Engine](https://www.dynatrace.com/support/help/shortlink/deploy-k8)  
+- [Metrics] [deploy a Google Cloud Function](https://www.dynatrace.com/support/help/shortlink/deploy-gcp)
 
 
 ## Pricing
-Ingested metrics will consume DDUs. For more details [refer to documention](https://www.dynatrace.com/support/help/reference/monitoring-consumption-calculation/#expand-gcp-service-monitoring-consumption-104)
+Ingested metrics will consume DDUs. For more details [refer to documentation](https://www.dynatrace.com/support/help/reference/monitoring-consumption-calculation/#expand-gcp-service-monitoring-consumption-104)
```

Comment:
It's worth adding link to DDU pricing for logs https://www.dynatrace.com/support/help/shortlink/calculate-log-consumption